CodeChallenge/Challenges/LetterCombinationsOfPhoneNumber/Entries/juliand665LetterCombinationOfPhoneNumberEntry.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  juliand665LetterCombinationOfPhoneNumberEntry.swift
+//  CodeChallenge
+//
+//  Created by Julian Dunskus on 06.01.17.
+//  Copyright © 2017 iosdevelopers. All rights reserved.
+//
+
+import Foundation
+
+let juliand665LetterCombinationOfPhoneNumberEntry = CodeChallengeEntry<LetterCombinationsOfPhoneNumberChallenge>(name: "juliand665") { input in
+	
+	var combs: [String] = [""]
+	
+	let expanded = input.characters.flatMap { numbersToLetters[$0] }
+	
+	for letters in expanded {
+		combs = combs.flatMap { prev in
+			letters.map { prev.appending($0) }
+		}
+	}
+	
+	return combs
+}
+
+private let numbersToLetters: [Character: [String]] = [ // explicit declaration reduces compile time
```

Comment:
Actually I take that back... I always forget that `private` is _more_ restrictive than `fileprivate`, so nevermind! No changes needed :)